language/move-prover/bytecode/src/function_target_pipeline.rs

```diff
@@ -313,7 +313,18 @@ impl FunctionTargetPipeline {
         let mut dep_ordered = vec![];
         while !worklist.is_empty() {
             // Put functions with 0 calls first in line, at the end of the vector
-            worklist.sort_by(|(_, callees1), (_, callees2)| callees2.len().cmp(&callees1.len()));
+            worklist.sort_by(|(_, callees1), (caller2, callees2)| {
+                if Ordering::Equal == callees2.len().cmp(&callees1.len()) {
```

Comment:
Comment here explaining  what this is doing might be useful

Reply:
done